app/js/main.js

```diff
@@ -46,7 +50,13 @@ fileDrop.addEventListener(DRAG_DROP_EVENT, function (e) {
     fileDrop.classList.remove(DRAG_OVER_CSS);
 
     [...e.dataTransfer.files].forEach(function (file) {
-        if (null === file.type.match(IMAGE_MIME_TYPE_PATTERN)) {
+        if (!file.type.match(IMAGE_MIME_TYPE_PATTERN)) {
+            const errorMessage = 'Invalid file type. Only image files are allowed.';
+            console.error(errorMessage);
```

Comment:
Error message should reported in the UI not the console. As the error message is implemented below, this can be removed here.

Reply:
Please change

Reply:
Done